src/python/CMSMonitoring/datasourcesList.py

```diff
@@ -8,54 +8,70 @@
 import requests
 
 try:
-    import urllib.request as ulib # python 3.X
+    import urllib.request as ulib  # python 3.X
     import urllib.parse as parser
 except ImportError:
-    import urllib2 as ulib # python 2.X
+    import urllib2 as ulib  # python 2.X
     import urllib as parser
-    
-class OptionParser():
+
+
+class OptionParser:
     def __init__(self):
         "User based option parser"
         desc = """
 This app creates a json file with the name, id, and type of datasource in the user organization. 
 It requires a admin token from grafana. 
                """
-        self.parser = argparse.ArgumentParser(prog='PROG', usage=desc)
-        self.parser.add_argument("--token", action="store",
-            dest="token", default=None, help="Admin token")
-        self.parser.add_argument("--url", action="store",
-            dest="url", default="https://monit-grafana.cern.ch", help="MONIT URL")
-        self.parser.add_argument("--output", action="store",
-            dest="output", default=sys.stdout, help="output file")
-
-def get_datasources(token, base='https://monit-grafana.cern.ch'):
+        self.parser = argparse.ArgumentParser(prog="PROG", usage=desc)
```

Comment:
you need to define desc here. I suggest to put into description a note about all environment variables this script relies on along with option descriptions.

Reply:
It is defined, just doesn't change from the current version (I run black to format the code and it touch some lines for style fixes), but I agree that it should be more explanative and include the definition of the env vars. I added that in the 
845fa15  commit.